spring-cloud/spring-cloud-zuul/spring-zuul-ui/src/main/java/com/baeldung/spring/cloud/zuul/filter/CustomZuulErrorFilter.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.spring.cloud.zuul.filter;
+
+import java.net.ConnectException;
+import java.time.Instant;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.netflix.zuul.ZuulFilter;
+import com.netflix.zuul.context.RequestContext;
+import com.netflix.zuul.exception.ZuulException;
+
+@Component
+public class CustomZuulErrorFilter extends ZuulFilter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CustomZuulErrorFilter.class);
+
+    private static String RESPONSE_BODY = "{\n" + "    \"timestamp\": " + "\"" + Instant.now()
+        .toString() + "\"" + ",\n" + "    \"status\": 503,\n" + "    \"error\": \"Service Unavailable\"\n" + "}";
+
+    @Override
+    public Object run() {
+        RequestContext context = RequestContext.getCurrentContext();
+        Throwable throwable = context.getThrowable();
+
+        if (throwable instanceof ZuulException) {
+            final ZuulException zuulException = (ZuulException) throwable;
+            LOG.error("Zuul exception: " + zuulException.getMessage());
+
+            if (throwable.getCause()
+                .getCause()
+                .getCause() instanceof ConnectException) {
```

Comment:
Since the method call is repeated I think you should write it in just one line.

Reply:
@JavierPeris I did that in the article. But the formatter is formatting in this way. I corrected it anyways